tips/TIP-0025/core-rest-api.yaml

```diff
@@ -0,0 +1,3329 @@
+      properties:
+        name:
+          type: string
+          description: The name of the node.
+        version:
+          type: string
+          description: The semantic version of the node.
+        status:
+          description: Status of the node.
+          properties:
+            isHealthy:
+              type: boolean
+              description: Tells whether the node is healthy or not.
+            latestMilestone:
+              type: object
+              description: Information about the latest seen milestone.
+              properties:
+                index:
+                  type: integer
+                  description: The index of the latest seen milestone.
+                timestamp:
+                  type: integer
+                  description: The timestamp of the latest seen milestone.
+                milestoneId:
+                  type: string
+                  description: The Milestone ID of the latest seen milestone. Hex-encoded with 0x prefix.
+              required:
+                - index
+            confirmedMilestone:
+              type: object
+              description: Information about the latest confirmed milestone.
+              properties:
+                index:
+                  type: integer
+                  description: The index of the latest confirmed milestone.
+                timestamp:
+                  type: integer
+                  description: The timestamp of the latest confirmed milestone.
+                milestoneId:
+                  type: string
+                  description: The Milestone ID of the latest confirmed milestone. Hex-encoded with 0x prefix.
+              required:
+                - index
+          required:
+            - isHealthy
+            - latestMilestone
+            - confirmedMilestone
+        metrics:
+          description: Node metrics.
+          properties:
+            blocksPerSecond:
+              description: The current rate of new blocks per second.
+              type: number
+              format: float
+            referencedBlocksPerSecond:
+              description: The current rate of referenced blocks per second.
+              type: number
+              format: float
+            referencedRate:
+              description: The ratio of referenced blocks in relation to new blocks of the last confirmed milestone.
+              type: number
+              format: float
+          required:
+            - blocksPerSecond
+            - referencedBlocksPerSecond
+            - referencedRate
+        supportedProtocolVersions:
+          type: array
+          description: The supported protocol versions.
+          items:
+            type: integer
+        protocol:
+          description: Protocol parameters.
+          properties:
+            networkName:
+              type: string
+              description: The Name of the network from which the networkId is derived.
+            bech32HRP:
```

Comment:
Same discussion as for `minPoWScore`: https://github.com/iotaledger/tips/pull/57/files#r921955443

Reply:
Changed in https://github.com/iotaledger/tips/pull/57/commits/103cdfa643012c0120faf4088cf5cd745e1b5de2